text/0043-better-eviction.md

```diff
@@ -88,6 +88,13 @@ generate a random number between 0 and 8, where 0, 1, 2, 3 means Alice and 4, 5,
 Carol. We apply the PRNG again: This time it's 5 and the slot goes to Carol.
 She is this round's proposer.
 
+Thirdly, it doesn't make sense to start an era where _everyone_ is excluded.
+So:
+* If at the beginning of an era, everyone with nonzero weight is banned, un-ban
```

Comment:
Yes, we do; in https://github.com/casper-network/casper-node/pull/1404/commits/cf02a3229e67d808c0cff1d2635a74d3d1c0ffdd we un-ban if everyone is banned; otherwise we only un-exclude if everyone is excluded.